src/Storage/Storage.Management/StorageAccount/SetAzureStorageAccount.cs

```diff
@@ -325,7 +325,7 @@ public bool EnableActiveDirectoryDomainServicesForFile
             HelpMessage = "Specifies the domain GUID. This parameter must be set when -EnableActiveDirectoryDomainServicesForFile is set to true.",
             ParameterSetName = ActiveDirectoryDomainServicesForFileParameterSet)]
         [ValidateNotNullOrEmpty]
-        public string ActiveDirectoryDomainGuid { get; set; }
+        public Guid ActiveDirectoryDomainGuid { get; set; }
```

Comment:
this is a breaking change.
We should iether keep the input type as string, or follow the breaking process to release it.